src/main/scala/com/example/QuickstartServer.scala

```diff
@@ -0,0 +1,29 @@
+package com.example
+
+//#quick-start-server
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+
+import akka.actor.ActorRef
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Route
+import akka.stream.ActorMaterializer
+
+object QuickstartServer extends App with MyRoutes {
```

Comment:
after getting rid of Actor, this would look like:

```
object QuickstartServer extends App with MyRoutes with OKTable {

  // set up ActorSystem and other dependencies here
  implicit val system: ActorSystem = ActorSystem("helloAkkaHttpServer")
  implicit val materializer: ActorMaterializer = ActorMaterializer()

  override val lookup = 
    (key: String) => Future(localStore.get(key))(ExecutionContext.fromExecutor(Executors.newFixedThreadPool(32)))

  // from the MyRoutes trait
  lazy val routes: Route = MyRoutes

  Http().bindAndHandle(routes, "localhost", 8080)

  println(s"Server online at http://localhost:8080/")

  Await.result(system.whenTerminated, Duration.Inf)
}
```

Reply:
Haha answer sheet right there.  Thanks.